### PR TITLE
Add queries to link status

### PR DESCRIPTION
**Pull request by sangjinhan**
This is a C++ port of #109, although it has been million years late...

### CODE REVIEW

File: core/bessctl.cc [hunk 1]
```diff
@@ -908,6 +926,33 @@ class BESSControlImpl final : public BESSControl::Service {
 
     return Status::OK;
   }
+
+  Status GetLinkStatus(ServerContext*, const GetLinkStatusRequest* request,
+                       GetLinkStatusResponse* response) override {
+    const char* port_name;
+    ::Port::LinkStatus status;
+
+    if (!request->name().length())
```
Comment (barath) [inline core/bessctl.cc]: Braces.

File: protobuf/bess_msg.proto [hunk 2]
```diff
@@ -151,6 +151,18 @@ message ListWorkersResponse {
   repeated WorkerStatus workers_status = 2;
 }
 
+message GetLinkStatusRequest {
```
Comment (barath) [inline protobuf/bess_msg.proto]: Could you add protobuf comments that follow the style of Justine's?

Comment (sangjinhan) [inline protobuf/bess_msg.proto]: I believe the message name is self describing... Justine's comments are about module-specific messages (module_msg.proto), not system-wide messages.

**Comment by sangjinhan** [comment]
Applied Barath's comments. Did a little bit of code polishing as well.

File: protobuf/bess_msg.proto (hunk 2 above)
Comment (barath) [inline protobuf/bess_msg.proto]: Ok.  I could go either way on this.  In a narrow sense it is self descriptive but a) it's nice to have everything documented for consistency so no guesswork is required and the meaning of Link and Status aren't actually totally obvious in the BESS context.

**Comment by melvinw** [comment]
Thanks! cat

**Pull request merged**